### PR TITLE
feat: update cluster-patch script

### PR DESCRIPTION
**Pull request by vprashar2929**
This PR updates the cluster-patch script.

### CODE REVIEW

File: hack/cpaas/patch-cluster-with-rhpm-0.1.sh [hunk 1]
```diff
@@ -1,96 +1,64 @@
...
-  #info "username: $username, password: $password"
-
-  info "getting auth from cluster"
-  run oc get secret/pull-secret -n openshift-config -o json | jq -r '.data.".dockerconfigjson"' | base64 -d > authfile
-
-  info "logging to brew registry"
-  run podman login --authfile authfile --username "$username" --password "$password" brew.registry.redhat.io
-
-  info "set auth to cluster"
-  run oc set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson=authfile
```
Comment (vimalk78) [inline hack/cpaas/patch-cluster-with-rhpm-0.1.sh:82]: please add this function back

**Review by vimalk78 — CHANGES_REQUESTED** [review]

File: hack/cpaas/patch-cluster-with-rhpm-0.1.sh (hunk 1 above)
Comment (vprashar2929) [reply hack/cpaas/patch-cluster-with-rhpm-0.1.sh:82]: Without this I am able to install operator

Comment (vimalk78) [reply hack/cpaas/patch-cluster-with-rhpm-0.1.sh:82]: i am not able to :)

Comment (vprashar2929) [reply hack/cpaas/patch-cluster-with-rhpm-0.1.sh:82]: strange 🤔

Comment (vimalk78) [reply hack/cpaas/patch-cluster-with-rhpm-0.1.sh:82]: not strange. perhaps the  pull secret you use to create the cluster already has the brew registry token.
that function just pulls the token if it exists and uses it, else asks you to create and it and exits.
If cluster already has the same token in pull secret nothing will change. 
This script is intended for users who do not usually pull images from brew directly.

Comment (vprashar2929) [reply hack/cpaas/patch-cluster-with-rhpm-0.1.sh:82]: maybe the QE clusters already contain brew registry token. I will revert this change

File: hack/cpaas/patch-cluster-with-rhpm-0.1.sh [hunk 2]
```diff
@@ -1,96 +1,108 @@
...
+	return $ret
+
 }
 
+add_brew_registry() {
+	header "Getting credentials for brew registry"
+	local url="https://employee-token-manager.registry.redhat.com/v1/tokens"
+	local ret=0
+
+	! token=$(curl --negotiate -u : $url -s) || [[ -z $token ]] || [[ $token == "null" ]] && {
```
Comment (sthaha) [inline hack/cpaas/patch-cluster-with-rhpm-0.1.sh:74]: IMHO, this is overly terse for really no good reason. In other words .. it becomes hard to reason what this does.

Comment (vprashar2929) [reply hack/cpaas/patch-cluster-with-rhpm-0.1.sh:74]: Agree. I didn't want to play with the original script too much

**Pull request closed without merging**